### DISCUSSION TITLE
Issue with apt-get package

### DISCUSSION THREAD

**Discussion by Deepak-Beniya** [OE4T/tegra-demo-distro#299] [Q&A]
I have added sudo , apt-get, coreutils in the base image .
But tried to update or upgrade I am facing issue.
I have attached a picture here.
Need help how do i solve it.
![IMG_20240430_210801](https://github.com/OE4T/tegra-demo-distro/assets/125279401/33fd9308-cb8b-47f0-a5f3-d0ed73bfada1)

**Comment by madisongh** [comment]
I don't think there are too many folks around here that use runtime package management of any kind... a general web search of that error message might turn up something useful.
